sdk/src/main/java/com/qonversion/android/sdk/internal/QRemoteConfigManager.kt

```diff
@@ -89,7 +96,20 @@ internal class QRemoteConfigManager @Inject constructor(
                     }
 
                     override fun onError(error: QonversionError) {
-                        fireToCallbacks(contextKey) { onError(error) }
+                        val baseRemoteConfigList = fallbackData?.remoteConfigList ?: run {
+                            fireToCallbacks(contextKey) { onError(error) }
+                            return@onError
+                        }
+
+                        val remoteConfig = if (contextKey != null) {
+                            baseRemoteConfigList.remoteConfigForContextKey(contextKey)
+                        } else {
+                            baseRemoteConfigList.remoteConfigForEmptyContextKey
+                        }
```

Comment:
Small point - if we swap branches, we can get rid of `!` in condition (would be a but clearer)